tls-client/mbed_app.json

```diff
@@ -2,8 +2,36 @@
     "macros": [
         "MBEDTLS_USER_CONFIG_FILE=\"mbedtls_entropy_config.h\""
     ],
+    "config": {
+        "network-interface":{
+            "help": "options are ETHERNET,WIFI_ESP8266,WIFI_ODIN,WIFI_RTW,MESH_LOWPAN_ND,MESH_THREAD",
+            "value": "ETHERNET"
+        },
+        "esp8266-tx": {
+            "help": "Pin used as TX (connects to ESP8266 RX)",
+            "value": "PTD3"
+        },
+        "esp8266-rx": {
+            "help": "Pin used as RX (connects to ESP8266 TX)",
+            "value": "PTD2"
+        },
+        "esp8266-debug": {
+            "value": true
+        },
+        "wifi-ssid": {
+            "value": "\"SSID\""
+        },
+        "wifi-password": {
+            "value": "\"Password\""
+        }
+    },
     "target_overrides": {
-        "UBLOX_EVK_ODIN_W2": {
+        "*": {
+            "target.features_add": ["NANOSTACK"],
+            "platform.stdio-baud-rate": 115200,
```

Comment:
Please note that the default is 9600 and all our infrastructure (and examples) are set up to listen at 9600. Please revert this change, otherwise the tests will not pass.

Reply:
I think these tests would run faster with 115200? That's the quite standard baud rate for anything these days and all boards I'm aware of (admittedly that's not 100% of boards) - they all can do 115200. How difficult would it be to update the test infra?

But, I do get your point and updating the test infra + this change should be coordinated together.